.travis.yml

```diff
@@ -84,6 +84,15 @@ script:
     if [[ -v MAVEN_CHECKS ]]; then
       ./mvnw install -DskipTests -B -T C1 -P ci
     fi
+  - |
+    if [[ -v BUILD_PRESTO_DOCKER ]]; then
+      PRESTO_VERSION=$(./mvnw --quiet --batch-mode --non-recursive  exec:exec -Dexec.executable='echo' -Dexec.args='${project.version}')
+      docker build . --build-arg "PRESTO_VERSION=${PRESTO_VERSION}" -t "presto:${PRESTO_VERSION}"
+      CONTAINER_ID=$(docker run -d --rm presto:${PRESTO_VERSION})
+      RESULT=$(docker exec ${CONTAINER_ID} presto --execute "SELECT 1")
+      docker stop $CONTAINER_ID
+      [[ ${RESULT} == '"1"' ]]
```

Comment:
can you please extract this to separate script so we do not have to think how travis handle errors when some of previous lines fails

Reply:
That's a great idea

Reply:
Script extracted and travis now calls that.